entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityAccessControlHandler.kt

```diff
@@ -394,4 +400,82 @@ class EntityAccessControlHandler(
 
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build<String>()
     }
+
+    @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
```

Comment:
move it where it should logically be in the file (and not just at the end of the file)